src/main/java/ru/training/at/hw6/entities/User.java

```diff
@@ -0,0 +1,32 @@
+package ru.training.at.hw6.entities;
+
+import ru.training.at.hw6.utils.ConfigReader;
+
+public class User {
+
+    public static final User ROMAN = new User(ConfigReader.getProperty("name"),
```

Comment:
the field name is misleading, it is not always Roman, but depends on the properties